{{cookiecutter.project_slug}}/fabfile.py

```diff
@@ -369,6 +407,9 @@ def import_db(c, dump_file=None):
         hide="out",
     )
 
+    if bool(strtobool(str(with_media))):
```

Comment:
This was necessary with fabric v1 but fabric v2 already parses arguments for us (i.e. `with_media` is already a boolean).

Reply:
D'acc, OK pour corriger :+1: